virttest/utils_misc.py

```
@@ -2336,6 +2336,19 @@ def __get_unit_index(M):
         return ("%.20f" % data_size).rstrip("0")
 
 
+def is_wmic_available(session):
+    """
+    Check if 'wmic' is available on the Windows guest.
+    :param session: Windows guest session
+    :return: True if wmic can be executed, False otherwise
+    """
+    try:
+        status, _ = session.cmd_status_output("wmic /?", timeout=10)
+        return status == 0
```

Ruff (0.14.8)

Consider moving this statement to an `else` block

(TRY300)